macros/ServiceWorkerSidebar.ejs

```diff
@@ -51,54 +51,69 @@ var text = mdn.localStringMap({
 <section id="Quick_Links">
  <ol>
   <li><a href="/<%=locale%>/docs/Web/API/Service_Worker_API"><strong><%=text['Service_Worker_API']%></strong></a></li>
-  <li data-default-state="open"><a href="#"><strong><%=text['Service_Worker_guides']%></strong></a>
-   <ol>
-     <li><a href="/<%=locale%>/docs/Web/API/Service_Worker_API/Using_Service_Workers"><%=text['Using_Service_Workers']%></a></li>
-   </ol>
+  <li class="toggle">
```

Comment:
``{{ServiceWorkerSidebar}}`` is correctly converted 👍, as seen on https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API/Using_Service_Workers:

<img width="553" alt="pr 789 serviceworkersidebar" src="https://user-images.githubusercontent.com/286017/45390140-bb565100-b5e3-11e8-8139-4a883e703b01.png">